src/js/components/Search/SearchResults.js

```diff
@@ -15,13 +27,47 @@ const SearchResults = () => {
 
   return (
     <div style={wrapperStyle}>
-      <Card style={{ borderRadius: '10px' }}>
-        <Statistic
-          title="Count"
-          value={isValid ? (status === 'count' ? count : message) : '----'}
-          valueStyle={{ color: '#1890ff' }}
-          prefix={<TeamOutlined />}
-        />
+      <Card style={{ borderRadius: '10px', padding: '10px 33px', width: '900px' }}>
```

Comment:
Minor: the 900px width can be limiting on a phone screen... Maybe `max-width` would be more appropriate to address the issue of wide computer screens, and otherwise on narrower screen it should allow the content (i.e. the charts) to wrap.
But maybe this is limited by the `Col` component anyway...